### PR TITLE
Release PR for Patch release v1.9.1

### PR DESCRIPTION
**Pull request by laxmikantbpandhare**
**Description of the change:**

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [x] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Review by varshaprasad96 — COMMENTED** [review]
@laxmikantbpandhare can you run `make generate`. I would expect testdata/ to be updated.

File: changelog/generated/v1.9.1.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## v1.9.1
+
+### Bug Fixes
+
+- Fixed the `operator-sdk generate bundle` command to specify the right path of bundle metadata in bundle.Dcokerfile. ([#5035](https://github.com/operator-framework/operator-sdk/pull/5035))
+- For Helm-based operators, fixed release equality comparison such that number values are compared and not their types to avoid unnecessary reconciliations. ([#5046](https://github.com/operator-framework/operator-sdk/pull/5046))
```
Comment (estroz) [inline changelog/generated/v1.9.1.md:6]: The problem this fix attempts to resolve may still exist, and the original fix may be reverted. See https://github.com/operator-framework/operator-sdk/pull/5059#issuecomment-884336514. We may want to remove this change from v1.9.x until it gets resolved, so we don't have to do 2 reversions.

**Comment by laxmikantbpandhare** [comment]
> @laxmikantbpandhare can you run `make generate`. I would expect testdata/ to be updated.

@varshaprasad96 - I tried but did not get any testdata files

File: changelog/generated/v1.9.1.md (hunk 1 above)
Comment (estroz) [reply changelog/generated/v1.9.1.md:6]: On second thought, lets stick with the order of things in master.

File: changelog/generated/v1.9.1.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## v1.9.1
+
+### Bug Fixes
+
+- Fixed the `operator-sdk generate bundle` command to specify the right path of bundle metadata in bundle.Dcokerfile. ([#5035](https://github.com/operator-framework/operator-sdk/pull/5035))
```
Comment (estroz) [inline changelog/generated/v1.9.1.md:5]:
```suggestion
- Fixed the `operator-sdk generate bundle` command to specify the right path of bundle metadata in bundle.Dockerfile. ([#5035](https://github.com/operator-framework/operator-sdk/pull/5035))
```

File: changelog/generated/v1.9.1.md (hunk 1 above)
Comment (laxmikantbpandhare) [reply changelog/generated/v1.9.1.md:6]: ok

File: changelog/generated/v1.9.1.md (hunk 2 above)
Comment (laxmikantbpandhare) [reply changelog/generated/v1.9.1.md]: Modified.

File: changelog/generated/v1.9.1.md (hunk 1 above)
Comment (laxmikantbpandhare) [reply changelog/generated/v1.9.1.md:6]: @estroz Didn't get your point. Do we need to remove change for helm #5046 ?

**Comment by laxmikantbpandhare** [comment]
> @laxmikantbpandhare can you run `make generate`. I would expect testdata/ to be updated.

Finally, able to generate the testdata.

**Review by jmrodri — APPROVED** [review]
/lgtm

**Review by theishshah — APPROVED** [review]
/lgtm

**Pull request merged**